modules/book.js

```diff
@@ -0,0 +1,7 @@
+export default class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+}
```

Comment:
- [ ] Hi you really did an amazing job but as part of the requirements, you are supposed to use ES6 function components instead of classes. Please kindly change all your classes to ES6 arrow functions

Reply:
Hi @m05e5 

Thanks for your review, However, I believe using ES6 function components instead of classes not from the project requirement